cc33xx-fw/suite/trixie/debian/cc33xx-fw.install

```diff
@@ -0,0 +1 @@
+*.bin /lib/
```

Comment:
Should this be `/lib/firmware`?

Reply:
It should be `/lib/firmware/ti-connectivity/`. Fixed.